gradle/java-setup.gradle

```diff
@@ -38,7 +34,8 @@ eclipseResourceFilters {
 apply plugin: 'findbugs'
 findbugs {
 	toolVersion = VER_FINDBUGS
-	sourceSets = [sourceSets.main]	// don't check the test code
+	sourceSets = [
+		sourceSets.main]	// don't check the test code
```

Comment:
For array I have only a max length. For example with `groovy.formatter.longListLength=1`, we can assure that all entries are in separate lines. Hence we can archive:
`
	sourceSets = [
		// don't check the test code
		sourceSets.main
	]
`